### DISCUSSION TITLE
Problem accessing Copilot

### DISCUSSION THREAD

**Discussion by toninogioia** [community/community#145576] [GitHub Education]
### Hi everyone,

I recently got approved as faculty and got access to all the benefits (at least so it states in my profit). Yet, I can't access Copilot. I wonder if anyone else has the same problem. Relatedly, it seems that I can't submit any support tickets to the topic of Education so no help from GitHub there.

**Comment by davevad93** [comment]
Hi @toninogioia ,

What's the status of your application?

If the label is green and states "`Congrats! Your request for @toninogioia was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @toninogioia was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

**Reply by toninogioia** [reply]
Thanks! Just to clarify, I'm a verified faculty and my status says that all benefits are now available. When I follow your steps, I only see the option to start a free trial, but no separate button for a Teacher's benefit.

**Reply by davevad93** [reply]
Hey @toninogioia , that's odd, by clicking on the link you should be able to redeem your offer, these are the same steps I gave a couple of hours ago in the following discussion and it worked #145550 . Try going to your [copilot settings](https://github.com/settings/copilot) and check if you can access it, if you can't, you should contact support by opening a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue with `Copilot` they will reply as soon as possible.

**Reply by toninogioia** [reply]
Thanks. I submitted the ticket, but only received an autoreply to contact the offer's support. However, Copilot doesn't allow me to submit a ticket, only post here. I'll see, maybe they reply to my ticket.

**Reply by davevad93** [reply]
You're welcome @toninogioia , as a last resort, try accessing this [link](https://education.github.com/pack/redeem/copilot-student) to see if you can redeem it.

**Comment by BhathiyaPrasad** [comment]
GitHub copilot is actually not free for student developer pack it only provide us a free trail

**Reply by davevad93** [reply]
Hey @BhathiyaPrasad , `Copilot` is actually free with GitHub Education benefits, if you're still with the trial version, it's most likely because you haven't redeemed your offer on the [pack offers page](https://education.github.com/pack).

<img width="307" alt="0000" src="https://github.com/user-attachments/assets/ecfe3e1a-45a3-4f9f-ac52-d7890aa4621a">

**Reply by toninogioia** [reply]
Yes, I don't have this link. That's my issue.

**Reply by BhathiyaPrasad** [reply]
@davevad93 Thankyou Somuch for information it works now

**Reply by BhathiyaPrasad** [reply]
@toninogioia can you send a screenshot shot of GitHub education portal because i was waiting for 3 days to get my package after they confirmed my request

**Reply by davevad93** [reply]
You're welcome @BhathiyaPrasad 👍

**Reply by toninogioia** [reply]
This is the screenshot of my GitHub Education:
<img width="341" alt="Screenshot 2024-11-25 at 13 59 00" src="https://github.com/user-attachments/assets/5e01b291-bfd1-47d8-b6ef-f9cc944bfa00">

and this is what I see when I follow the link 
<img width="419" alt="Screenshot 2024-11-25 at 14 01 04" src="https://github.com/user-attachments/assets/fc3da4fd-fe0f-42ea-82ed-b3e113d1dd48">

**Reply by BhathiyaPrasad** [reply]
@toninogioia ah that's the case it will take 1 to 3 days to process your benefits keep calm and wait it will appear soon

**Reply by toninogioia** [reply]
Gotcha! Thanks!
